src/test/resources/features/convertImage/image.feature

```diff
@@ -0,0 +1,283 @@
+      | height           |  10                               |
+      | width            |  10                               |
+      | gray             |  on                               |
+      | extractMetadata  |  on                               |
+      | extractThumbnail |  on                               |
+    Then I validate the response has the "400" status code
+    And I validate that the response body should match with "common/errorResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 400                 |
+      | error   | "Failed in the md5" |
+
+
+  @negative
+  Scenario: Verify that the position field does not allow data that is not of type (x,y)
+    When I send a POST request to "/convertImage" with the following form data
+      | file             |  @"template/image/image.jpg"      |
+      | md5              |  1589d7f466220b7604858b1eaabdd0d2 |
+      | exportFormat     |  .png                             |
+      | name             |  test                             |
+      | changeSize       |  on                               |
+      | position         |  2                                |
+      | height           |  10                               |
+      | width            |  10                               |
+      | gray             |  on                               |
+      | extractMetadata  |  on                               |
+      | extractThumbnail |  on                               |
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "common/errorResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 400                                |
+      | error   | "The position field is type (x,y)" |
+
+  @functional
+  Scenario: Verify that without the changeSize field the height and width fields are not required
+    When I send a POST request to "/convertImage" with the following form data
+      | file             |  @"template/image/image.jpg"      |
+      | md5              |  1589d7f466220b7604858b1eaabdd0d2 |
+      | exportFormat     |  .png                             |
+      | name             |  test                             |
+      | changeSize       |  off                              |
+      | position         |  2                                |
+      | height           |                                   |
+      | width            |                                   |
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "common/messageResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 200                               |
+
+  @negative
+  Scenario: Verify that the numeric fields do not accept negative numbers
```

Comment:
Please consider improving this scenario title to something like: `Verify that is not possible to perform image conversion with invalid image dimensions`